src/main/java/com/afornalik/ox/ui/UI.java

```diff
@@ -11,6 +13,10 @@ public UI(UIOutputDestination uiOutputDestination, UIDrawBoard uiDrawBoard) {
     }
 
     public void printBoard() {
-        uiOutputDestination.print(uiDrawBoard.draw());
+        try {
+            uiOutputDestination.print(uiDrawBoard.draw());
+        } catch (OutOfBoardException e) {
+            e.printStackTrace();
+        }
```

Comment:
Same as above

Reply:
same - replace with custom message from exception.

Reply:
Same as above

Reply:
also will be replaced.